### PR TITLE
Markdown: Add check if ID is less than 17 characters in length

### PR DESCRIPTION
**Pull request by auravoid**
because github is dumb sometimes, all the commits from #4 are also here :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This is the other way to get the bot into one of your servers, and is best for d
 - Clone the repo
 - Copy `config.example.json` to `config.json`
 - In `config.json`, provide values for at least `botToken`, `homeChannel` and `homeServerId`. Other values are optional, but some functionality may not work without them (the bot should tell you what's missing though if you try to use a feature that requires a value you didn't set). If you're not sure about a value, see [the wiki page on config.json](https://github.com/FloatingMilkshake/MechanicalMilkshake/wiki/Configuration#configjson) or feel free to contact me!
-- Run `docker-compose up -d`
+- Run `docker-compose up -d`
```
Comment (FloatingMilkshake) [inline README.md:45]: huh?

Comment (auravoid) [reply README.md:45]: as the commit says, dwbi.

Comment (FloatingMilkshake) [reply README.md:45]: I'm not much of a fan of pointless commits... what was this for?

Comment (auravoid) [reply README.md:45]: It was to see if I could do a squash to my repo that would remove all of #4's commits from this PR but it didn't work lmao

Comment (FloatingMilkshake) [reply README.md:45]: fair enough

File: README.md [hunk 2]
```diff
@@ -45,4 +45,4 @@ This is the other way to get the bot into one of your servers, and is best for d
 - Run `docker-compose up -d`
 
 ## Contributing
-I'm not the best at this, so feel free to open an issue or PR if you notice anything that seems wrong or if you have a suggestion! I'm all ears. However, please note that I have some [Contribution Guidelines](CONTRIBUTING.md).
\ No newline at end of file
+I'm not the best at this, so feel free to open an issue or PR if you notice anything that seems wrong or if you have a suggestion! I'm all ears. However, please note that I have some [Contribution Guidelines](CONTRIBUTING.md).
```
Comment (FloatingMilkshake) [inline README.md]: alright sure github

Comment (auravoid) [reply README.md]: **_yes_**

**Review by FloatingMilkshake — APPROVED** [review]

**Pull request merged**